### PR TITLE
fix(app): 6.0 overall design QA

### LINKED ISSUE #11076: 6.0 Feedback:  Overall Design QA
### Overview

must-address feedback for 6.0

### Current Behavior

**T1 Protocols Page**
- [x] (1) Missing hover state on protocol card
- [x] (2) Fix spacing issue on the card. Please refer to original designs. 
- [x] (3) Fix spacing on open menu

Screen Shot 2022-07-11 at 2 34 40 PM

**app-wide**
- [x] (1) Navigation hover has been fixed on edge, push this to 6.0. Basically the hover colour is off, shouldn't be grey. 
- [x] (2) Can we close the menu on click? [Loom showing current state] This is on a lot of the buttons. (https://www.loom.com/share/42e678efaf204b8a83f79f54b0ba557e) - This is fixed everywhere except the Robot Settings Calibration which will be fixed by my PR: https://github.com/Opentrons/opentrons/pull/11020

Screen Shot 2022-07-11 at 2 35 06 PM

**T1 Labware Page**
- [x] (1) Get open menu in appropriate position (not cutting off screen)
- [x] (2) Add hover state to the card
- [x] (3) If the copy button has no interactivity to inform the user they've copied the line, then we should nix this button entirely
- [x] (4) Wrap text so users don't see this
Screen Shot 2022-07-11 at 2 35 55 PM
Screen Shot 2022-07-11 at 2 36 00 PM

**T2 Devices**
- [x] (1) Users shouldn't see an empty space
Screen Shot 2022-07-11 at 2 36 36 PM

**T3 Robot Settings + T1 App Settings**
- [x] (1) Can we bump up the size of the toggles that are smaller to match the larger grey one? 
Screen Shot 2022-07-11 at 2 37 20 PM

**T4 Protocol Run**
- [x] (1) Update the hover state on this button. shouldn't go to blue 
- [x] (2) This shouldn't happen! 

Screen Shot 2022-07-11 at 2 37 46 PM
Screen Shot 2022-07-11 at 2 37 50 PM

### Expected Behavior

_No response_

### Steps To Reproduce

_No response_

### Operating system

_No response_

### Robot setup or anything else?

_No response_

### PR DESCRIPTION
**Pull request by vabruzzo**
# Overview
Addresses various design QA items for 6.0
Closes #11076.

# Changelog
- refer to ticket for list of styling edits
- removed copy link button on labware card

# Review requests
Check various QA items.

# Risk assessment
Low

### CODE REVIEW

**Review by koji — APPROVED** [review]
Looks good to me!

**Review by jerader — APPROVED** [review]
lgtm! your PR title is wrong though? Should be `fix(app): 6.0 overall design QA`?

**Comment by vabruzzo** [comment]
> lgtm! your PR title is wrong though? Should be `fix(app): 6.0 overall design QA`?

woops, been here 2 months and just make this mistake now... thanks for catching!

**Comment by mmencarelli** [comment]
- [x] (1) This is not a blocker, but just want to call out that the colours here still aren't matching. *minor, not a blocker
- [ ] (2) This one is still smaller! *minor, not a blocker
- [x] (3) Blocker: Module card shouldn't be full width of two columns

Screen Shot 2022-07-12 at 10 59 12 AM
Screen Shot 2022-07-12 at 10 59 08 AM
Screen Shot 2022-07-12 at 10 59 03 AM

File: components/src/icons/icon-data.ts [hunk 1]
```diff
@@ -440,9 +440,9 @@ export const ICON_DATA_BY_NAME = {
       'M15.5 0h-9C2.9 0 0 2.9 0 6.5S2.9 13 6.5 13h9c3.6 0 6.5-2.9 6.5-6.5S19.1 0 15.5 0zm-9 11C4 11 2 9 2 6.5S4 2 6.5 2 11 4 11 6.5 9 11 6.5 11z',
   },
   'ot-toggle-input-on': {
-    viewBox: '0 0 22 14',
+    viewBox: '0 0 22 13',
```
Comment (jerader) [inline components/src/icons/icon-data.ts:443]: whoops, i thought changing the `viewBox` fixed this! Thanks for adding a new icon

**Review by jerader — APPROVED** [review]
lgtm! thanks for adding the new icon and fixing the module cards in module controls

File: components/src/icons/icon-data.ts (hunk 1 above)
Comment (vabruzzo) [reply components/src/icons/icon-data.ts:443]: it did  kinda fix it, but it caused a 1 or 2 pixel cutoff of the bottom of the icon. there is probably some way to edit the path slightly to get it to work but exporting a new icon was easier, something was a bit off with this one i think

**Pull request merged**